<?xml version="1.0"?>
<launch>

  <arg name="move_gripper_debug"		default="false"/>
  <group ns="px150">


    <node name="move_gripper"
      pkg="flex_grasp"
      type="move_gripper.py"
      output="screen">
      <param name ="debug" value="$(arg move_gripper_debug)"/>
    </node>
  </group>
</launch>
